File: AED/Laravel/RestauranteRevised/app/Models/Mesa.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Mesa extends Model
{
    //use HasFactory;
    private int $num_mesa;
    private int $sillas;

    public function __construct($num_mesa = 0, $sillas = 0 ){
        $this->num_mesa = $num_mesa;
        $this->sillas = $sillas;
    }


    /**
     * Get the value of num_mesa
     */ 
    public function getNum_mesa()
    {
        return $this->num_mesa;
    }

    /**
     * Set the value of num_mesa
     *
     * @return  self
     */ 
    public function setNum_mesa($num_mesa)
    {
        $this->num_mesa = $num_mesa;

        return $this;
    }

    /**
     * Get the value of sillas
     */ 
    public function getSillas()
    {
        return $this->sillas;
    }

    /**
     * Set the value of sillas
     *
     * @return  self
     */ 
    public function setSillas($sillas)
    {
        $this->sillas = $sillas;

        return $this;
    }

   
}

File: AED/JPA/reservas/src/main/java/es/iespuertodelacruz/saul/entities/Mesa.java
package es.iespuertodelacruz.saul.entities;

import java.io.Serializable;
import javax.persistence.*;
import java.util.List;


/**
 * The persistent class for the mesas database table.
 * 
 */
@Entity
@Table(name="mesas")
@NamedQuery(name="Mesa.findAll", query="SELECT m FROM Mesa m")
public class Mesa implements Serializable {
	private static final long serialVersionUID = 1L;

	@Id
	@GeneratedValue(strategy=GenerationType.IDENTITY)
	@Column(name="num_mesa")
	private int numMesa;

	private int sillas;

	//bi-directional many-to-one association to Reserva
	@OneToMany(mappedBy="mesa")
	private List<Reserva> reservas;

	public Mesa(int numMesa, int sillas) {
		super();
		this.numMesa = numMesa;
		this.sillas = sillas;
	}

	public Mesa() {
	}

	public int getNumMesa() {
		return this.numMesa;
	}

	public void setNumMesa(int numMesa) {
		this.numMesa = numMesa;
	}

	public int getSillas() {
		return this.sillas;
	}

	public void setSillas(int sillas) {
		this.sillas = sillas;
	}

	public List<Reserva> getReservas() {
		return this.reservas;
	}

	public void setReservas(List<Reserva> reservas) {
		this.reservas = reservas;
	}

	public Reserva addReserva(Reserva reserva) {
		getReservas().add(reserva);
		reserva.setMesa(this);

		return reserva;
	}

	public Reserva removeReserva(Reserva reserva) {
		getReservas().remove(reserva);
		reserva.setMesa(null);

		return reserva;
	}

}